### PR TITLE
Merge 4.43 code freeze to develop

### PR DESCRIPTION
**Pull request by oguzkocer**

### CODE REVIEW

File: Simplenote/Resources/AppStoreStrings.pot [hunk 1]
```diff
@@ -15,7 +15,8 @@ msgstr ""
 
 #. translators: Subtitle to be displayed below the application name in the Apple App Store. Limit to 30 characters including spaces and commas!
 msgctxt "app_store_subtitle"
-msgid "Simple notes, todos and memos"
+msgid "Simple notes, todos and memos
+"
```
Comment (AliSoftware) [inline Simplenote/Resources/AppStoreStrings.pot]: This bug from our toolkit that generates the `.pot` strikes again 😒 

```suggestion
msgid "Simple notes, todos and memos"
```

File: Simplenote/Resources/AppStoreStrings.pot [hunk 2]
```diff
@@ -63,12 +64,14 @@ msgstr ""
 #. translators: Keywords used in the App Store search engine to find the app.
 #. Delimit with an English comma between each keyword. Limit to 100 characters including spaces and commas.
 msgctxt "app_store_keywords"
-msgid "notes,text,planner,journal,diary,sync,to-do,cloud,notebook,simple,notepad,tag,todo,writing,memo"
+msgid "notes,text,planner,journal,diary,sync,to-do,cloud,notebook,simple,notepad,tag,todo,writing,memo
+"
```
Comment (AliSoftware) [inline Simplenote/Resources/AppStoreStrings.pot]: ditto here

```suggestion
msgid "notes,text,planner,journal,diary,sync,to-do,cloud,notebook,simple,notepad,tag,todo,writing,memo"
```

**Review by AliSoftware — DISMISSED** [review]

**Comment by AliSoftware** [comment]
PS: As a workaround for next time, I wonder if, until we find some time to properly fix it in the toolkit, removing the trailing `\n` from the corresponding `…/source/{keywords,subtitle}.txt` files could help avoid this bug for next releases in the meantime?

File: Simplenote/Resources/AppStoreStrings.pot (hunk 2 above)
Comment (mokagio) [reply Simplenote/Resources/AppStoreStrings.pot]: I took a stab at fixing this issue here https://github.com/wordpress-mobile/release-toolkit/pull/297.

**Review by mokagio — COMMENTED** [review]
- [x] New version header in `RELEASE-NOTE.txt` 
- [x] Version update in `.xcconfig`, `Deliverfile`, and `download_metadata.swift` 
- [x] `Localizable.strings` updated 
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version 
- [x] `AppStoreStrings.pot` updated accordingly to new version and release notes

**Comment by mokagio** [comment]
I pushed 57f7bb1 to address the `.pot` issue and reviewed the PR.

I'm going to dismiss @AliSoftware 's request for changes to let the auto-merge kick in so the new strings can get to GlotPress.

**Review by mokagio — APPROVED** [review]

**Review by AliSoftware — APPROVED** [review]

**Pull request merged**